examples/06_scope_functions/05_also.md

```diff
@@ -0,0 +1,32 @@
+# also
+
+`also` works quite like [`apply`](04_apply.md): it executes a given block and returns the called object.
+Inside the block, the object is referenced by `it`, so it's easier to pass it as an agrument.
+This function is handy for embedding side effects, such as logging, in call chains. 
```

Comment:
This function is handy for embedding side effects, such as logging in call chains. 

This function is handy for embedding side effects, such as logging, and in call chains. 


'embedding side effects' -- side effects are normally something negative.
alts:
side benefits
secondary effects